index.js

```diff
@@ -1,11 +1,22 @@
 var DtsCreator = require('typed-css-modules');
-
-var creator = new DtsCreator();
+var loaderUtils = require('loader-utils');
+var objectAssign = require('object-assign');
 
 module.exports = function(source, map) {
   this.cacheable && this.cacheable();
   var callback = this.async();
-  
+
+  // Pass on query parameters as an options object to the DtsCreator. This lets
+  // you change the default options of the DtsCreator and e.g. use a different
+  // output folder.
+  var queryOptions = loaderUtils.parseQuery(this.query);
+  var options;
```

Comment:
I've tested it btw, since in the end it turned out I didn't need to pass any options to the loader, thus having an undefined options object.